### PR TITLE
Add openstack repos mirror

### PR DESCRIPTION
**Pull request by dulek**

### CODE REVIEW

**Comment by dulek** [comment]
@smarterclayton: Seems like this does work, at least when I tried applying that base-openstack-4-2 Service manually. I'm not sure if adding the secret creation to prow-ocp-rpm-secrets step in the Makefile is enough to trigger its creation in the CI infra?

**Comment by dulek** [comment]
@stevekuznetsov: I don't have an idea how to check that and honestly I don't think I'd have enough privileges on the cluster to try it out.

File: ci-operator/infra/openshift/release-controller/repos/ocp-4.3-openstack.repo [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[rhel-openstack]
+name = rhel-openstack
+baseurl = https://use-mirror1.ops.rhcloud.com/enterprise/reposync/4.1/rhel-openstack-rpms/
+          https://use-mirror2.ops.rhcloud.com/enterprise/reposync/4.1/rhel-openstack-rpms/
```
Comment (smarterclayton) [inline ci-operator/infra/openshift/release-controller/repos/ocp-4.3-openstack.repo]: Why are these configured for reposing 4.1?

File: ci-operator/infra/openshift/release-controller/repos/ocp-4.2-openstack.repo [hunk 2]
```diff
@@ -0,0 +1,10 @@
+[rhel-openstack]
+name = rhel-openstack
+baseurl = https://use-mirror1.ops.rhcloud.com/enterprise/reposync/4.1/rhel-openstack-rpms/
+          https://use-mirror2.ops.rhcloud.com/enterprise/reposync/4.1/rhel-openstack-rpms/
```
Comment (smarterclayton) [inline ci-operator/infra/openshift/release-controller/repos/ocp-4.2-openstack.repo]: this should be `4.2` - if you aren't mirroring to 4.2, we need to get that fixed

Comment (dulek) [reply ci-operator/infra/openshift/release-controller/repos/ocp-4.2-openstack.repo]: Here's the reason: https://github.com/openshift/release/blob/d6ea197bb29b214f3ba5b6eca8826db378340a81/ci-operator/infra/openshift/release-controller/repos/ocp-4.2-default.repo#L3-L4. I assumed ocp-4.2.default.repo got it figured out.

Comment (dulek) [reply ci-operator/infra/openshift/release-controller/repos/ocp-4.2-openstack.repo]: Should I still change it?

**Comment by elfosardo** [comment]
@smarterclayton hi!
This change is also very important, if not fundamental, for testing the ironic images (we have 7 so far, almost all of them depend on openstack packages); currently two requests for CI configuration, #4348 and #4349, are blocked because we can't access openstack packages.
Please let me know if there's anything we could do to help merging this change, thanks!

File: ci-operator/infra/openshift/release-controller/repos/ocp-4.2-openstack.repo (hunk 2 above)
Comment (smarterclayton) [reply ci-operator/infra/openshift/release-controller/repos/ocp-4.2-openstack.repo]: Can you open a bug to the ART team that they need to sync 4.2 repos to mirror.openshift.com?

**Comment by smarterclayton** [comment]
/lgtm

Follow up on 4.2

**Pull request merged**